src/features/encoders/mozJPEG/shared/meta.ts

```diff
@@ -37,3 +37,67 @@ export const defaultOptions: EncodeOptions = {
   separate_chroma_quality: false,
   chroma_quality: 75,
 };
+
+/**
+ * Simple helper function that wraps each line of the input at `line_width` characters (default 60)
+ */
+function wrapText(text: string, line_width: number = 60): string {
+  const lines = text.split('\n');
+  let balancedText = '';
+  for (const line of lines) {
+    balancedText += '\n';
+    const words = line.split(' ');
+    let runningLength = 0;
+    for (const word of words) {
+      if (runningLength + word.length + 1 > line_width) {
+        balancedText += '\n' + word + ' ';
+        runningLength = word.length;
+      } else {
+        runningLength += word.length + 1;
+        balancedText += word + ' ';
+      }
+    }
+  }
+  return balancedText.trim();
+}
+
+// Until we make a better way of displaying information (like with modals), displaying a simple 'title' attribute should suffice for each codec parameter
+type mozjpeg_param = keyof typeof defaultOptions;
+const paramDescriptions: Record<mozjpeg_param, string> = {
```

Comment:
With the logic above removed, this file can be a singular default export of this dictionary.